saleor/graphql/account/mutations.py

```diff
@@ -148,6 +152,29 @@ class Meta:
         model = models.User
 
 
+class LoggedUserUpdate(CustomerCreate):
+    class Arguments:
+        input = UserAddressInput(
+            description='Fields required to update logged in customer.',
+            required=True)
+
+    class Meta:
+        description = 'Updates data of the logged in user.'
+        exclude = ['password']
+        model = models.User
+
+    @classmethod
+    def user_is_allowed(cls, user, input):
+        id = graphene.Node.to_global_id('User', user.id)
+        return user.has_perm('account.manage_users') or id == input['id']
```

Comment:
So I think this function is inaccurate at this moment. No matter of user's permission, one can update his own data. I think we should just check if user is logged in.